### PR TITLE
Added ⚡️ content to `Getting Started` > `Why bitcoin is unique`

### PR DESCRIPTION
**Pull request by GBKS**
Added a small paragraph about extensibility to hint at the Lightning network.
Added links to other pages as appropriate.
Revised visual styling of images as they were just black and white before.

As this is a pretty general page about Bitcoin, I don't think it is necessary to add more about Lightning, beyond this one point about extensibility.

[🧞‍♂️ Check the preview 🧚‍♀️](https://deploy-preview-477--sad-borg-390916.netlify.app/guide/getting-started/why-bitcoin-is-unique/)

Addresses #463.

### CODE REVIEW

**Review by Bosch-0 — DISMISSED** [review]
LGTM :)

File: guide/getting-started/why-bitcoin-is-unique.md [hunk 1]
```diff
@@ -67,15 +67,17 @@ In addition, six fundamental properties define whether or not a currency can act
 5. Divisibility
 6. Recognizability
 
-Although bitcoin is purely digital, it meets every classical definition of what makes something money. Instead of relying on physical properties (like gold and silver) or central authorities (like government-issued fiat currencies), bitcoin relies on the world’s most powerful computer network to mathematically enforce the rules that make it the first truly digital form of cash.
+Although bitcoin is purely digital, it meets every classical definition of what makes something money. Instead of relying on physical properties (like gold and silver) or central authorities (like government-issued fiat currencies), bitcoin relies on the world’s most powerful [computer network]({{ '/guide/getting-started/technology-primer/#what-is-a-node' | relative_url }}) to mathematically enforce the rules that make it the first truly digital form of cash.
+
+Natively digital money can also be extended in previously unthinkable ways. From smart contracts to Layer 2 solutions like the Lightning Network, Bitcoin can get built upon to enable new and evolving use cases.
```
Comment (danielnordh) [inline guide/getting-started/why-bitcoin-is-unique.md:72]:
```suggestion
Natively digital money can also be extended in previously unthinkable ways. From smart contracts to Layer 2 solutions like the Lightning Network that enable faster and cheaper payments, Bitcoin can get built upon to enable new and evolving use cases.
```

Comment (danielnordh) [reply guide/getting-started/why-bitcoin-is-unique.md:72]: Suggest we hint at what this is good for as the terms (Layer 2, Lightning network) aren't self explanatory.

**Comment by pavlenex** [comment]
@GBKS you think we can tackle https://github.com/BitcoinDesign/Guide/issues/433 in this one as well?

**Comment by GBKS** [comment]
@pavlenex good idea, I added a bit about the supply.

**Review by pavlenex — DISMISSED** [review]
LGTM

**Comment by pavlenex** [comment]
@GBKS Actually the img doesn't load

**Review by pavlenex — APPROVED** [review]

**Pull request merged**